Update documenting a feature for a release doc (#25199)
kubernetes/website: content/en/docs/contribute/new-content/new-features.md

### Open a placeholder PR

> 1. Open a **draft** pull request against the

Reviewer: "draft" here looks like a relative term to "ready for review" below, if I'm understanding this correctly.

Please be specific how to mark a PR as a "draft". Do you mean prefixing the PR title with "WIP -" or something?

Author: It's a stage of a pull request that is offered by Github. I can see that it may be not clear, so I added steps on how to mark a pull request as "draft" and "ready for review" in the corresponding sections. Could you take a look again? Does this make more sense?

### Open a placeholder PR

1. Open a **draft** pull request against the
`dev-{{< skew nextMinorVersion >}}` branch in the `kubernetes/website` repository, with a small
commit that you will amend later. To create a draft pull request, use the
Create Pull Request drop-down and select **Create Draft Pull Request**,

> then click **Draft Pull Request**.

Reviewer: Okay, now I see. Draft Pull Request is something one can raise from Web UI. If I'm opening a PR from git command line, there is actually no difference between a "draft" and a "formal" one, right?

Author: Yes, that's correct. This is specific to Github not git.
It should not impact the people using the git command line to make a pull request, they'll have to use the UI to set it to draft if needed.

Reviewer: Thanks for the clarification. @annajung

Reviewer 2: I think the GitHub command line tools do let you mark a PR as draft, and also to mark it ready for review: [`gh pr create --draft`](https://cli.github.com/manual/gh_pr_create).

### PR ready for review

When ready, populate your placeholder PR with feature documentation and change
the state of the PR from draft to **ready for review**.

Do your best to describe your feature and how to use it. If you need help structuring your documentation, ask in the `#sig-docs` slack channel.

When you complete your content, the documentation person assigned to your feature reviews it.
To ensure technical accuracy, the content may also require a technical review from corresponding SIG(s).
Use their suggestions to get the content to a release ready state.

If your feature is an Alpha or Beta feature and is behind a feature gate,
make sure you add it to [Alpha/Beta Feature gates](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features)

> table as part of your pull request. With Alpha, a description of the feature gate

Reviewer: ```suggestion
table as part of your pull request. With new feature gates, a description of the feature gate
```

> is a also required. If your feature is moving out of Beta,

Reviewer: ```suggestion
is also required. If your feature is GA'ed or deprecated,
```
